src/main/java/turniplabs/bhopboots/BhopBoots.java

```diff
@@ -18,9 +19,17 @@ public static String name(String name) {
         return BhopBoots.MOD_ID + "." + name;
     }
 
+    static {
+        try {
+            Class.forName("net.minecraft.core.block.Block");
+            Class.forName("net.minecraft.core.item.Item");
+        } catch (ClassNotFoundException ignored) {
+        }
+    }
```

Comment:
What's this for?

Reply:
I had a crash with registry item thought this would of fixed it I just forgot to remove and test it .